server/base/src/main/java/org/apache/accumulo/server/tables/TableManager.java

```diff
@@ -115,10 +115,21 @@ public TableState getTableState(String tableId) {
 
     final TableState oldState;
     final TableState newState;
+    final String message;
 
     public IllegalTableTransitionException(TableState oldState, TableState newState) {
+      this(oldState, newState, "");
+    }
+
+    public IllegalTableTransitionException(TableState oldState, TableState newState, String message) {
       this.oldState = oldState;
       this.newState = newState;
+
+      String defaultMessage = "Error transitioning from " + oldState + " state to " + newState + " state";
```

Comment:
Could this be moved inside the else block?

Reply:
Yes...done.